core/platform/image/gae_image_services.py

```diff
@@ -67,7 +65,10 @@ def compress_image(image_content, scaling_factor):
         str. Returns the content of the compressed image.
```

Comment:
Is line 58 above still relevant if we're moving fully to Pillow? We might still want to bound the dimensions anyway, but the reasoning might need to be updated.

Reply:
Ah I should have checked the reasoning for the bounding. I just checked the Pillow documentation and it no longer has that limit so I will remove it.